examples/frompapers/Brunel_Wang_2001.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
+
+'''
+Sample-specific persistent activity
+-----------------------------------
+
+Five subpopulations with three selective and one reset stimuli example.
+Analog to figure 6b in the paper.
+
+BRUNEL, Nicolas et WANG, Xiao-Jing. Effects of neuromodulation in a cortical network model of object working memory 
+dominated by recurrent inhibition. Journal of computational neuroscience, 2001, vol. 11, no 1, p. 63-85.
+'''
+
+from brian2 import *
+
+# populations
+N = 1000
+N_E = int(N * 0.8)  # pyramidal neurons
+N_I = int(N * 0.2)  # interneurons
+
+# voltage
+V_L = -70. * mV
+V_thr = -50. * mV
+V_reset = -55. * mV
+V_E = 0. * mV
+V_I = -70. * mV
+
+# membrane capacitance
+C_m_E = 0.5 * nF
+C_m_I = 0.2 * nF
+
+# membrane leak
+g_m_E = 25. * nS
+g_m_I = 20. * nS
+
+# refractory period
+tau_rp_E = 2. * ms
+tau_rp_I = 1. * ms
+
+# external stimuli
+rate = 3 * Hz
+C_ext = 800
+
+# synapses
+C_E = N_E
+C_I = N_I
+
+# AMPA (excitatory)
+g_AMPA_ext_E = 2.08 * nS
+g_AMPA_rec_E = 0.104 * nS * 800. / N_E
+g_AMPA_ext_I = 1.62 * nS
+g_AMPA_rec_I = 0.081 * nS * 800. / N_E
+tau_AMPA = 2. * ms
+
+# NMDA (excitatory)
+g_NMDA_E = 0.327 * nS * 800. / N_E
+g_NMDA_I = 0.258 * nS * 800. / N_E
+tau_NMDA_rise = 2. * ms
+tau_NMDA_decay = 100. * ms
+alpha = 0.5 / ms
+Mg2 = 1.
+
+# GABAergic (inhibitory)
+g_GABA_E = 1.25 * nS * 200. / N_I
+g_GABA_I = 0.973 * nS * 200. / N_I
+tau_GABA = 10. * ms
+
+# subpopulations
+f = 0.1
+p = 5
+N_sub = int(N_E * f)
+N_non = int(N_E * (1. - f * p))
+w_plus = 2.1
+w_minus = 1. - f * (w_plus - 1.) / (1. - f)
+
+# modeling
+eqs_E = '''
+dv / dt = (- g_m_E * (v - V_L) - I_syn) / C_m_E : volt (unless refractory)
+
+I_syn = I_AMPA_ext + I_AMPA_rec + I_NMDA_rec + I_GABA_rec : amp
+
+I_AMPA_ext = g_AMPA_ext_E * (v - V_E) * s_AMPA_ext : amp
+I_AMPA_rec = g_AMPA_rec_E * (v - V_E) * 1 * s_AMPA : amp
+ds_AMPA_ext / dt = - s_AMPA_ext / tau_AMPA : 1
+ds_AMPA / dt = - s_AMPA / tau_AMPA : 1
+
+I_NMDA_rec = g_NMDA_E * (v - V_E) / (1 + Mg2 * exp(-0.062 * v / mV) / 3.57) * s_NMDA_tot : amp
+s_NMDA_tot : 1
+
+I_GABA_rec = g_GABA_E * (v - V_I) * s_GABA : amp
+ds_GABA / dt = - s_GABA / tau_GABA : 1
+'''
+
+eqs_I = '''
+dv / dt = (- g_m_I * (v - V_L) - I_syn) / C_m_I : volt (unless refractory)
+
+I_syn = I_AMPA_ext + I_AMPA_rec + I_NMDA_rec + I_GABA_rec : amp
+
+I_AMPA_ext = g_AMPA_ext_I * (v - V_E) * s_AMPA_ext : amp
+I_AMPA_rec = g_AMPA_rec_I * (v - V_E) * 1 * s_AMPA : amp
+ds_AMPA_ext / dt = - s_AMPA_ext / tau_AMPA : 1
+ds_AMPA / dt = - s_AMPA / tau_AMPA : 1
+
+I_NMDA_rec = g_NMDA_I * (v - V_E) / (1 + Mg2 * exp(-0.062 * v / mV) / 3.57) * s_NMDA_tot : amp
+s_NMDA_tot : 1
+
+I_GABA_rec = g_GABA_I * (v - V_I) * s_GABA : amp
+ds_GABA / dt = - s_GABA / tau_GABA : 1
+'''
+
+P_E = NeuronGroup(N_E, eqs_E, threshold='v > V_thr', reset='v = V_reset', refractory=tau_rp_E)
```

Comment:
Could you add a `method=...` argument (e.g. `method='euler') here? We try to avoid that examples show any INFO or WARNING messages.